flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSourceITCase.java

```diff
@@ -591,8 +591,7 @@ public void testStreamPartitionReadByPartitionName() throws Exception {
 
             Assert.assertEquals(
                     Arrays.asList(
-                            Row.of(i, "new_add", 11 + i, "2020", "10", "0" + i).toString(),
-                            Row.of(i, "new_add_1", 11 + i, "2020", "10", "0" + i).toString()),
+                            Row.of(i, "new_add").toString(), Row.of(i, "new_add_1").toString()),
```

Comment:
Looks like the we missed one line here?

Reply:
Hello, it not missed. As we have changed to `select x,y... where ..` ,  So we only assertEquals the column x, y. The `Row.of(i, "new_add").toString(), Row.of(i, "new_add_1").toString()),` is 2 line.